src/components/Glossary/GlossaryTooltip/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactNode } from "react"
+import { Text, useBreakpointValue } from "@chakra-ui/react"
+
+import GlossaryDefinition from "../GlossaryDefinition"
+import Tooltip from "../../Tooltip"
+
+interface IProps {
+  children: ReactNode
+  termKey: string
+}
+
+const GlossaryTooltip: React.FC<IProps> = ({ children, termKey }) => {
+  const isLargeScreen = useBreakpointValue({ base: false, lg: true })
+
+  return isLargeScreen ? (
+    <Tooltip content={<GlossaryDefinition term={termKey} size="sm" />}>
+      <Text
+        as="u"
+        textDecorationStyle="dotted"
+        _hover={{
+          textDecorationColor: "primary.hover",
+          color: "primary.hover",
+        }}
+        cursor="help"
+      >
+        {children}
+      </Text>
+    </Tooltip>
+  ) : (
+    <Text>{children}</Text>
```

Comment:
On small resolutions, this will create a `p` tag which is a block element and will create some problems like:
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/4d19b520-2e3c-4f62-92ad-8238c2ff8c29)
(the text `dilemma` is the one wrapped with the tooltip)

Proposal: use an inline element
```suggestion
    <Text as="span">{children}</Text>
```